### ISSUE TITLE
create Github issues on record submit

### ISSUE THREAD

**Issue by n-a-t-e** [cioos-siooc/metadata-entry-form#196]
This is something that Hakai will test out, when a record is submitted it will create an issue in an issue-only github repo via firebase actions. This would simplify tracking the progress of datasets, and outsource some functionality from the metadata form. Once the issue is created, the rest of the process is done entirely in github. The second iteration of this could add github usernames so that certain reviewers could be alerted when there's a new record for review that falls under their projects (See #191)

Similar idea to the http://github.com/cioos-siooc/cwatch-upptime repo where its more about assigning tasks then editing code/text files.

**Comment by n-a-t-e** [comment]
Will submit a new issue with Dataset Title as the title, link to the metadata record in the form, and the text included here:
https://github.com/HakaiInstitute/metadata-review/edit/main/.github/ISSUE_TEMPLATE/dataset-name.md

**Comment by n-a-t-e** [comment]
Fixed by #210
